js/components/options_input.js

```diff
@@ -41,5 +44,13 @@ export default {
     valid: function () {
       return this._isValid(this.value)
     },
+    showError: function () {
+      const showError =
+        (this.initialErrors && this.initialErrors.length) || false
```

Comment:
```suggestion
        this.initialErrors && this.initialErrors.length
```
I think this line can be simplified. LGTM other than that